src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -29,25 +28,26 @@
   final HeaderView headerView;
   private final Rscc model;
   private final Strings strings = new Strings();
-  final Label keyGenerationLbl = new Label();
-  final Label supporterAdminLbl = new Label();
 
-  VBox mainBox = new VBox();
+  final Label titleLbl = new Label();
+  final Label predefinedAddressesLbl = new Label();
+  final Label descriptionLbl = new Label();
+
+  final VBox descriptionBox = new VBox();
   final VBox bottomBox = new VBox();
 
   final HBox supporterAdminBox = new HBox();
-  final VBox centerBox = new VBox();
+  final HBox centerBox = new HBox();
   final HBox keyGeneratingBox = new HBox();
 
   final TitledPane keyGeneratorPane = new TitledPane();
-  final TitledPane supporterAdminPane = new TitledPane();
-
-  final Text descriptionTxt = new Text();
-  final Text additionalDescriptionTxt = new Text();
+  final TitledPane predefinedAddressesPane = new TitledPane();
 
   final TextField generatedKeyFld = new TextField();
 
   final Button reloadKeyBtn = new Button();
+  final Button readyBtn = new Button();
```

Comment:
That's a really good question - as far as I understood from @pwigger it should show if it's possible to connect or not. However, we could realise this with a label or so.

Maybe we should bring it up on the next team meeting